go/Makefile

```diff
@@ -1,5 +1,5 @@
 test:
-	go test .
+	go test -short .
```

Comment:
Why add the flag? Even if we don't skip the tests, it doesn't take that long to run them.

Reply:
Cool, I've reverted it, thanks for raising that.